### PR TITLE
mutagen: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/mutagen.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# mutagen
+
+> Real-time file synchronization and network forwarding tool.
+> More information: <https://mutagen.io/>.
```
Comment (bl-ue) [inline pages/common/mutagen.md:4]:
```suggestion
> More information: <https://mutagen.io>.
```

File: pages/common/mutagen.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start a synchronization session between a local directory and a remote host:
+
+`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{user@example.com:/path/to/remote/directory/}}`
```
Comment (bl-ue) [inline pages/common/mutagen.md:8]: `host` is more explicit and readable here than `example.com`, IMO
```suggestion
`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{user@host:/path/to/remote/directory/}}`
```

**Review by bl-ue — APPROVED** [review]

File: pages/common/mutagen.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# mutagen
+
+> Real-time file synchronization and network forwarding tool.
+> More information: <https://mutagen.io>.
+
+- Start a synchronization session between a local directory and a remote host:
+
+`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{user@host:/path/to/remote/directory/}}`
```
Comment (navarroaxel) [inline pages/common/mutagen.md:8]: Based on the SSH pages, the variables are the following:
```suggestion
`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{user}}@{{host}}:{{/path/to/remote/directory/}}`
```

File: pages/common/mutagen.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start a synchronization session between a local directory and a Docker container:
+
+`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{docker://user@container_name/path/to/remote/directory/}}`
```
Comment (navarroaxel) [inline pages/common/mutagen.md:12]:
```suggestion
`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} docker://{{user}}@{{container_name/path/to/remote/directory/}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/mutagen.md (hunk 3 above)
Comment (marchersimon) [inline pages/common/mutagen.md:8]:
```suggestion
`mutagen sync create --name={{session_name}} {{/path/to/local/directory/}} {{user@host:/path/to/remote/directory/}}`
```

File: pages/common/mutagen.md (hunk 4 above)
Comment (marchersimon) [inline pages/common/mutagen.md:12]:
```suggestion
`mutagen sync create --name={{session_name}} {{/path/to/local/directory/}} {{docker://user@container_name/path/to/remote/directory/}}`
```

File: pages/common/mutagen.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Stop a running session:
+
+`mutagen sync terminate {{session-name}}`
```
Comment (marchersimon) [inline pages/common/mutagen.md:16]:
```suggestion
`mutagen sync terminate {{session_name}}`
```

**Review by marchersimon — CHANGES_REQUESTED** [review]

File: pages/common/mutagen.md (hunk 4 above)
Comment (Rudloff) [reply pages/common/mutagen.md]: Maybe separate the container name and the path then?
```bash
mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} docker://{{user}}@{{container_name}}{{/path/to/remote/directory/}}
```

**Review by sbrl — APPROVED** [review]
Thanks, @Rudloff! Looks fine to me.

Just a question though. How does the user id mapping work when you're syncing with a docker container?

**Review by marchersimon — APPROVED** [review]

**Comment by Rudloff** [comment]
> How does the user id mapping work when you're syncing with a docker container?

I think that mutagen does not keep file ownership, the synced files will be owned by the user specified in the `mutagen sync create` argument.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**